presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -1210,6 +1211,15 @@ public Void visitLateralJoin(LateralJoinNode node, Void context)
             return processChildren(node, context);
         }
 
+        @Override
+        public Void visitNativeExecution(NativeExecutionNode node, Void context)
```

Comment:
can you share somewhere an example of what this will actually look like when it gets printed.

Reply:
+1